google/auth/external_account.py

```diff
@@ -147,7 +147,12 @@ def __init__(
         super(Credentials, self).__init__()
         self._audience = audience
         self._subject_token_type = subject_token_type
+        self._universe_domain = universe_domain or credentials.DEFAULT_UNIVERSE_DOMAIN
```

Comment:
Is this already available from the parent class?

Reply:
The defaulting is, but it doesn't set the universe_domain from the constructor.